asconfig.json

```diff
@@ -37,7 +37,7 @@
             },
             {
                 "name": "CONFIG::timeStamp",
-                "value": "\"2020-03-28\""
+                "value": "\"9999-12-31\""
```

Comment:
What's with this value?

Reply:
Development only, so I figured it doesn't matter and the more outrageous the less confusing it would be. Otherwise we have to keep two version locations up to date. Agree/Disagree?

Reply:
So this is used to indicate a dev-only build. :+1: